database/table/resources.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Nooku Framework - http://nooku.org/framework
+ *
+ * @copyright   Copyright (C) 2011 - 2015 Johan Janssens and Timble CVBA. (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        http://github.com/joomlatools/joomlatools-framework-activities for the canonical source repository
+ */
+
+/**
+ * Resources Database Table.
+ *
+ * @author  Arunas Mazeika <https://github.com/amazeika>
+ * @package Koowa\Component\Activities
+ */
+class ComActivitiesDatabaseTableResources extends KDatabaseTableAbstract
+{
+    /**
+     * Initializes the options for the object.
+     *
+     * Called from {@link __construct()} as a first step of object instantiation.
+     *
+     * @param KObjectConfig $config Configuration options.
+     */
+    protected function _initialize(KObjectConfig $config)
+    {
+        $config->append(array(
+            'name'      => 'resources',
+            'behaviors' => array(
+                'com:activities.database.behavior.resources.creatable',
+                'parameterizable' => array('column' => 'metadata')
+            ),
+            'filters'   => array(
+                'metadata' => 'json'
```

Comment:
@amazeika  In TEXTman we are calling this column data, this column will contain the entity data right, metadata would imply it's data about the data?